src/pages/__tests__/testTeams.tsx

```diff
@@ -29,7 +29,31 @@ describe('Teams', () => {
     });
 
     it('should render spinner while loading', async () => {
-        // TODO - Add code for this test
+        jest.spyOn(API, 'getTeams').mockImplementation(() => new Promise((resolve) => setTimeout(() => resolve([
```

Comment:
Test on render spinner, with a mocked time running so that we test when it should display and also if it disappears when needed